mvp-spec/ad-auction.md

```diff
@@ -1,25 +1,117 @@
-# Publishers requirements
+# Ad auction
 
 ## Goal of the document
 
-This document describes what is required for a Publisher to participate in the Prebid Addressability Framework (PAF).
+This document describes the Prebid Addressability Framework (PAF) requirements related to an ad auction.
 
 ## Overview
 
-The Prebid Addressability Framework enhances ad slots selling by instantiating an Audit Log. This Audit Log requires the following elements, that Publishers must implement:
-* a **Seed**, identifying a bid request for an ad slot for a given user
-* **Transmission Requests**, identifying the Seed and the requester at each step of the bidding workflow
-* **Transmission Responses**, as answers to Transmission Requests
-* The **Audit Log** itself, identifying the entities have been involved in the display of an ad
+The Prebid Addressability Framework enhances ad auctions by instantiating an **Audit Log**, built upon the following elements:
+* **Seed**
+* **Transmission Requests**
+* **Transmission Responses**
+* **Transmission Results**
 
-Seeds, Transmission Requests, and Transmissions Responses are signed.
+### Seed
 
-Publishers are also required to:
-* Expose an Identity endpoint, so that participants can verify the the publisher's signature.
+A Seed links together the publisher site identity and a transaction id. 
```

Comment:
"many transaction ids"

The new version of the specification states that an Addressable Content has a Transaction Id. A Seed is created for one or many Addressable Contents. Therefore, there are one or many Transaction Ids per Seed.

Reply:
Ok, fixed